### ISSUE TITLE
Add ASETAvionics

### ISSUE THREAD

**Issue by StoneBlue** [KSP-CKAN/NetKAN#9566]
```yaml
spec_version: v1.27
identifier: ASETAvionics
name: ASET Consolidated Avionics Pack
abstract: >-
  A props pack consisting of mostly Aircraft-style IVA props, controls,
  instruments & gauges, both "modern" & "classic" in style.
author:
  - alexustas
  - StoneBlue
  - JonnyOThan
license: CC-BY-NC-SA-3.0
download: https://spacedock.info/mod/3239/ASET%20Consolidated%20Avionics%20Pack
version: 3.0.0
$vref: '#/ckan/ksp-avc'
resources:
  bugtracker: https://github.com/StoneBlue/ASET-Consolidated-Props/issues
  spacedock: https://spacedock.info/mod/3239/ASET%20Consolidated%20Avionics%20Pack
  homepage: >-
    http://forum.kerbhttps://forum.kerbalspaceprogram.com/index.php?/topic/211905-1125-181-aset-consolidated-avionics-props-iva-packs-adopted/
  repository: https://github.com/StoneBlue/ASET-Consolidated-Avionics
  x_screenshot: >-
    https://spacedock.info/content/alexustas_3789/ASET_Avionics/ASET_Avionics-1486632235.7713456.png
tags:
  - library
  - crewed
ksp_version_min: 1.8.1
ksp_version_max: 1.12.99
depends:
  - name: ASETAgency
  - name: ASETProps
    min_version: 2.0.0
  - name: ModuleManager
  - name: RasterPropMonitor-Core
x_via: Generated by Metadata Webtool

```

**Comment by StoneBlue** [comment]
please use this PR in place of https://github.com/KSP-CKAN/NetKAN/pull/9558 <--close that one

Sorry

**Comment by HebaruSan** [comment]
Thanks, this has been merged into #9558 (the metadata-webtool submits issues, not pull requests, and we do still need a pull request).
